src/Microsoft.AspNetCore.Authentication/RemoteAuthenticationHandler.cs

```diff
@@ -69,7 +68,7 @@ public virtual Task<bool> ShouldHandleRequestAsync()
                 {
                     return true;
                 }
-                else if (authResult.Nothing)
+                else if (authResult.Skipped || authResult.Nothing)
```

Comment:
The old `AuthenticateResult.None()/Nothing` was split into `AuthenticateResult.None()/Nothing` (= no authentication ticket because authentication was skipped) and `RemoteAuthenticateResult.Skip()/Skipped` (no authentication ticket because one of the events opted for "skipping to next middleware).

We need to check both here.